### PR TITLE
LPS-203766 - Replace direct usage of ClayBadge with FeatureIndicator.

### PR DESCRIPTION
**Pull request by ethib137**
https://liferay.atlassian.net/browse/LPS-203766

This PR replaces the direct usage of the ClayBadge with the new FeatureIndicator component. 

@liferay-echo
Screenshot 2023-12-27 at 11 56 04 AM

@liferay-publications
Screenshot 2023-12-27 at 11 58 24 AM
Screenshot 2023-12-27 at 11 56 38 AM

### CODE REVIEW

**Comment by ethib137** [comment]
ci:test:sf

**Comment by ethib137** [comment]
cc: @ealonso @zsigmondrab

File: modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_info_collection_providers.jsp [hunk 1]
```diff
@@ -42,10 +42,16 @@ InfoCollectionProviderDisplayContext infoCollectionProviderDisplayContext = (Inf
 					<%= HtmlUtil.escape(infoCollectionProviderDisplayContext.getTitle(infoCollectionProvider)) %>
 
 					<c:if test="<%= infoCollectionProvider instanceof BetaInfoCollectionProvider %>">
-						<clay:label
-							displayType="info"
-							label='<%= LanguageUtil.get(request, "beta") %>'
-						/>
+						<span>
+							<react:component
```
Comment (ealonso) [inline modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_info_collection_providers.jsp:46]: @ethib137 Could be possible to create a clay taglib for it?

Comment (ealonso) [reply modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_info_collection_providers.jsp:46]: Having this kind of logic around portal is not easy to maintain and reuse.

Comment (ethib137) [reply modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_info_collection_providers.jsp:46]: Hey @ealonso  Yeah, I totally agree and we will. Once we create this we will also migrate all these instances.

I've created this story to track it: https://liferay.atlassian.net/browse/LPS-204695

Comment (ealonso) [reply modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_info_collection_providers.jsp:46]: Great!! 👏🏾

**Comment by ethib137** [comment]
ci:test:relevant

**Comment by ethib137** [comment]
ci:reevaluate:1297401_9007

**Comment by ethib137** [comment]
The failures are unrelated. I'm going to force forward.

**Comment by ethib137** [comment]
ci:forward:force

**Pull request closed without merging**